client/cli/go/cmds/cluster.go

```diff
@@ -27,24 +27,24 @@ import (
 )
 
 func init() {
-	RootCmd.AddCommand(clusterCommand)
-	clusterCommand.AddCommand(clusterCreateCommand)
-	clusterCommand.AddCommand(clusterDeleteCommand)
-	clusterCommand.AddCommand(clusterListCommand)
-	clusterCommand.AddCommand(clusterInfoCommand)
-	clusterCreateCommand.SilenceUsage = true
-	clusterDeleteCommand.SilenceUsage = true
-	clusterInfoCommand.SilenceUsage = true
-	clusterListCommand.SilenceUsage = true
+	RootCmd.AddCommand(ClusterCommand)
+	ClusterCommand.AddCommand(ClusterCreateCommand)
```

Comment:
You placed the test files in a different directory which forced you to export these values.  Please see other test examples.  All test are in the same directory as the program files so that they can access all unexported functions and members of a structure.